tests/README.md

```diff
@@ -0,0 +1,63 @@
+# Tests
+
+These tests should ideally be run for every Pull Request, though due to the need to run `clasp login` before some commands, some tests are not run.
+
+
+# Configuration using Travis
+
+This is a bit more difficult, and why we include the `.clasprc.json.enc` file. The `.clasprc.json` file is used to authenticate many of the `clasp` commands, and is what is generated after using `clasp login`.
+
+Here it is encrypted as `.clasprc.json.enc` and decrypted by Travis with the line:
+
+```openssl aes-256-cbc -K $encrypted_0f9bbf7a60f4_key -iv $encrypted_0f9bbf7a60f4_iv -in .clasprc.json.enc -out .clasprc.json -d || true```
+
+The way that Travis works is that encrypted files cannot be decrypted by a Pull Request from a fork (see: https://docs.travis-ci.com/user/encrypting-files/). There are ways around this, that may or may not be what we will do in the future, see (https://blog.algolia.com/travis-encrypted-variables-external-contributions/) for ideas.
+
+This is why the end of that command ends with ` || true` (so Travis doesn't fail on any PR).
```

Comment:
This is why the end of that command ends with ` || true` (so Travis doesn't fail on any PR).
->
Thus, commands end with ` || true` so Travis doesn't fail on any PR.

Reply:
👍